packages/zapp/console/src/components/common/Icons/RerunIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { IconProps } from './interface';
+
+export const RerunIcon: React.FC<IconProps> = ({ size = 18, className, onClick }) => {
```

Comment:
Please replace React.FC<> with a function as  `= () =>`.
React.FC has t's own downsides and planned to be deprecated. We should avoid using it.  Here is some explanation why: https://github.com/facebook/create-react-app/pull/8177
```
export const RerunIcon = (props: SVGProps): JSX.Element => (
<svg>...</svg>
)
```
If you feel like it `JSX.Element` return type can be replaced with more broader `React.ReactNode one`.

Reply:
Please replace it in all 4 places you were adding it and ensure to use props declaration instead of inline "name extraction" (not sure how it called in english).

I know it often seems redundant at first, but props is better coding style, simplifies future modification of props and allow easier inheritance of components or types in future. 
```
interface SomeProps {
 prop1: prop1type; ...
}
const Some = (props: SomeProps) => {...}
```

Reply:
fixed